src/huggingface_hub/inference/_common.py

```diff
@@ -65,6 +66,29 @@
 logger = logging.getLogger(__name__)
 
 
+# Add dataclass for ModelStatus. We use this dataclass in get_model_status function.
+@dataclass
+class ModelStatus:
+    """
+        This Dataclass represents the the model status in the Hugging Face Inference API.
+
+        Attributes
```

Comment:
```suggestion
        Args:
```

Just to be compliant with the doc builder :)